### ISSUE TITLE
Exporting with scale gives strange labels (they are shifted)

### ISSUE THREAD

**Issue by d0x** [highslide-software/highcharts.com#4618]
When i'm using an own exporting server, the scaling option breaks some labels. See:
![image](https://cloud.githubusercontent.com/assets/2202723/10247295/60e0135c-6918-11e5-851f-c93f728db045.png)

The same request send to export.highcharts.com looks fine. See:
![chart 35](https://cloud.githubusercontent.com/assets/2202723/10247438/6299ba3a-6919-11e5-9319-aa33bc078b54.png)

I used the following configuration (on AWS EC2):
* Ubuntu 14.04.3 LTS 
* oracle Java 1.8 or 1.7, both doesn't work
* Phantomjs 1.9

I used this script to setup the server:

```
# more or less from: http://www.highcharts.com/docs/export-module/setting-up-the-server
sudo apt-get update
sudo apt-get upgrade
sudo apt-get install maven phantomjs tomcat7 git -y
rm -rf highcharts.com
git clone https://github.com/highslide-software/highcharts.com.git
cd highcharts.com
git checkout tags/v4.1.8
cd ..
cd highcharts.com/exporting-server/java/highcharts-export/highcharts-export-web/
echo -e "\nexec=/usr/bin/phantomjs" >> src/main/webapp/WEB-INF/spring/app-convert.properties
cd ..
mvn package
sudo cp highcharts-export-web/target/highcharts-export-web.war /var/lib/tomcat7/webapps/
sudo service tomcat7 restart
```

As long as my testserver runs you can try it:

[*click* (link with data to my server '.../?scale=**2**&options=...', buggy)](http://54.93.204.252:8080/highcharts-export-web/?scale=2&options=%7B%22chart%22:%7B%22type%22:%22pie%22%7D,%22title%22:%7B%22text%22:%22%22%7D,%22series%22:%5B%7B%22size%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22SWMS%22,%22y%22:7.499999523162842,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Grundversorger%22,%22y%22:50.39999771118164,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22News%20%26%20Info%22,%22y%22:17.700000762939453,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22%C3%96koanbieter%22,%22y%22:4.400000095367432,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Online%20Disc%22,%22y%22:59.79999923706055,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Vergleicher%22,%22y%22:160.3000030517578,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%205%20?%20this.point.name%20:%20null;%20%7D,%22color%22:%22%23ffffff%22,%22distance%22:-40%7D%7D,%7B%22size%22:%2280%25%22,%22innerSize%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22PC%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:2.3,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22PC%22,%22y%22:26.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.8,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Phone%22,%22y%22:9.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22PC%22,%22y%22:5.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:5.3,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Phone%22,%22y%22:7.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22PC%22,%22y%22:1.2,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:1.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22PC%22,%22y%22:18.9,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.7,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:26.2,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22PC%22,%22y%22:45.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:61.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Phone%22,%22y%22:52.7,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%201%20?%20'%3Cb%3E'%20%2B%20this.point.name%20%2B%20':%3C/b%3E%20'%20%2B%20this.y%20%2B%20'%25'%20:%20null;%20%7D%7D%7D%5D,%22credits%22:%7B%22enabled%22:false%7D%7D)

[*click* (link with data to my server '.../?scale=**1**&options=...', fine)](http://54.93.204.252:8080/highcharts-export-web/?scale=2&options=%7B%22chart%22:%7B%22type%22:%22pie%22%7D,%22title%22:%7B%22text%22:%22%22%7D,%22series%22:%5B%7B%22size%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22SWMS%22,%22y%22:7.499999523162842,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Grundversorger%22,%22y%22:50.39999771118164,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22News%20%26%20Info%22,%22y%22:17.700000762939453,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22%C3%96koanbieter%22,%22y%22:4.400000095367432,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Online%20Disc%22,%22y%22:59.79999923706055,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Vergleicher%22,%22y%22:160.3000030517578,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%205%20?%20this.point.name%20:%20null;%20%7D,%22color%22:%22%23ffffff%22,%22distance%22:-40%7D%7D,%7B%22size%22:%2280%25%22,%22innerSize%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22PC%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:2.3,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22PC%22,%22y%22:26.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.8,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Phone%22,%22y%22:9.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22PC%22,%22y%22:5.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:5.3,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Phone%22,%22y%22:7.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22PC%22,%22y%22:1.2,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:1.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22PC%22,%22y%22:18.9,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.7,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:26.2,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22PC%22,%22y%22:45.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:61.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Phone%22,%22y%22:52.7,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%201%20?%20'%3Cb%3E'%20%2B%20this.point.name%20%2B%20':%3C/b%3E%20'%20%2B%20this.y%20%2B%20'%25'%20:%20null;%20%7D%7D%7D%5D,%22credits%22:%7B%22enabled%22:false%7D%7D)

[*click* (link with data to '...export.highcharts.com/?scale=**2**&options=...', fine)](http://export.highcharts.com/?scale=1&options=%7B%22chart%22:%7B%22type%22:%22pie%22%7D,%22title%22:%7B%22text%22:%22%22%7D,%22series%22:%5B%7B%22size%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22SWMS%22,%22y%22:7.499999523162842,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Grundversorger%22,%22y%22:50.39999771118164,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22News%20%26%20Info%22,%22y%22:17.700000762939453,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22%C3%96koanbieter%22,%22y%22:4.400000095367432,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Online%20Disc%22,%22y%22:59.79999923706055,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Vergleicher%22,%22y%22:160.3000030517578,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%205%20?%20this.point.name%20:%20null;%20%7D,%22color%22:%22%23ffffff%22,%22distance%22:-40%7D%7D,%7B%22size%22:%2280%25%22,%22innerSize%22:%2260%25%22,%22data%22:%5B%7B%22name%22:%22PC%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:2.3,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.6,%22color%22:%22%23003380%22%7D,%7B%22name%22:%22PC%22,%22y%22:26.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.8,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22Phone%22,%22y%22:9.3,%22color%22:%22%230075dc%22%7D,%7B%22name%22:%22PC%22,%22y%22:5.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:5.3,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22Phone%22,%22y%22:7.2,%22color%22:%22%2300998f%22%7D,%7B%22name%22:%22PC%22,%22y%22:1.2,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:1.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:2.1,%22color%22:%22%239dcc00%22%7D,%7B%22name%22:%22PC%22,%22y%22:18.9,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:14.7,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22Phone%22,%22y%22:26.2,%22color%22:%22%23ffff00%22%7D,%7B%22name%22:%22PC%22,%22y%22:45.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Tablet%22,%22y%22:61.8,%22color%22:%22%23ffa405%22%7D,%7B%22name%22:%22Phone%22,%22y%22:52.7,%22color%22:%22%23ffa405%22%7D%5D,%22dataLabels%22:%7B%22formatter%22:function%20()%20%7B%20return%20this.y%20%3E%201%20?%20'%3Cb%3E'%20%2B%20this.point.name%20%2B%20':%3C/b%3E%20'%20%2B%20this.y%20%2B%20'%25'%20:%20null;%20%7D%7D%7D%5D,%22credits%22:%7B%22enabled%22:false%7D%7D)

**Comment by gvaartjes** [comment]
Hi, 

This is a known issue and is fixed with installation of Phantomjs 2.0. Unfortunately there's still no official release for Linux. You have to build it yourself. http://phantomjs.org/build.html

Gert
Highsoft

**Comment by d0x** [comment]
It would be nice when mention that here: http://www.highcharts.com/docs/export-module/setting-up-the-server

Thank you.
